corehq/apps/sms/tests/test_backends.py

```diff
@@ -37,127 +39,145 @@
 from urllib import urlencode
 
 
-class AllBackendTest(BaseSMSTest):
+class AllBackendTest(DomainSubscriptionMixin, TestCase):
 
-    def setUp(self):
-        super(AllBackendTest, self).setUp()
-
-        self.domain_obj = Domain(name='all-backend-test')
-        self.domain_obj.save()
-        self.create_account_and_subscription(self.domain_obj.name)
-        self.domain_obj = Domain.get(self.domain_obj.get_id)
+    @classmethod
+    def setUpClass(cls):
```

Comment:
ah didn't realize it was required. fixed here: https://github.com/dimagi/commcare-hq/pull/15149/commits/37ca44397450734dab3b19d091ebcabbc98d0fee